src/data/graphql/queries.graphql

```diff
@@ -242,6 +242,7 @@ query ColonyAction($transactionHash: String!, $colonyAddress: String!) {
     domainName
     domainPurpose
     domainColor
+    motionId
```

Comment:
As I've mentioned in the previous PR review, you don't need this here, you already have in the `events` object!